cf_xarray/tests/test_accessor.py

```diff
@@ -119,7 +119,8 @@ def test_rename_like():
     (
         ("resample", {"time": "M"}, {"T": "M"}),
         ("rolling", {"lat": 5}, {"Y": 5}),
-        ("groupby", {"group": "time"}, {"group": "T"}),
+        ("groupby", {"group": "time"}, {"group": "T"}), 
+        ("groupby_time_accessor", {"group": "time.month"}, {"group": "T.month"}),
```

Comment:
```suggestion
        ("groupby", {"group": "time.month"}, {"group": "T.month"}),
```

the first element is `attr` which is the name of the function called `getattr(obj, attr)